sandbox/registration-processor-camel-routes-lost-mz.xml

```diff
@@ -191,18 +197,18 @@
       <from uri="eventbus://bio-dedupe-lost-bus-out" />
       <log message="bio-dedupe-->uin-generation lost route ${bodyAs(String)}" />
       <choice>
-         <when>
+        <!-- <when>
             <jsonpath>$.[?(@['isValid'] == false &amp;&amp; @['internalError'] == true)]</jsonpath>
             <to uri="workflow-cmd://complete-as-failed" />
-         </when>
+         </when>-->
          <when>
             <jsonpath>$.[?(@['isValid'] == true &amp;&amp; @['internalError'] == true)]</jsonpath>
             <to uri="workflow-cmd://mark-as-reprocess" />
          </when>
-         <when>
+        <!-- <when>
             <jsonpath>$.[?(@['isValid'] == false &amp;&amp; @['internalError'] == false)]</jsonpath>
             <to uri="workflow-cmd://complete-as-rejected" />
-         </when>
+         </when>-->
```

Comment:
this and above when commenting related to all biodedup stages to be removed, since after the rebasing the code from develop this commenting will not be required

Reply:
updated

Reply:
resolved